src/spread/sngl-obj-ident.case

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2017 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: Object Spread operator without other arguments
+template: default
+esid: pending
+includes: [propertyHelper.js]
+info: |
+    Pending Runtime Semantics: PropertyDefinitionEvaluation
+
+    PropertyDefinition:...AssignmentExpression
+
+    1. Let exprValue be the result of evaluating AssignmentExpression.
+    2. Let fromValue be GetValue(exprValue).
+    3. ReturnIfAbrupt(fromValue).
+    4. Let excludedNames be a new empty List.
+    5. Return CopyDataProperties(object, fromValue, excludedNames).
+
+---*/
+
+//- setup
+let o = {c: 3, d: 4};
```

Comment:
it would be nice to add some _false(ish)_ values: `e: undefined, f: null, g: false` and other non regular values as well: `h: -0, i: Symbol('tests')` and even a cross reference: `j: o`